--- xerces-c-3.2.3/configure.ac.old	2021-03-29 13:45:20.852389632 +1000
+++ xerces-c-3.2.3/configure.ac	2021-03-29 13:46:45.689641613 +1000
@@ -247,6 +247,10 @@
                     [
                       AC_MSG_RESULT([no])
                       AC_DEFINE_UNQUOTED([HAVE_WCSRTOMBS], 0, [Define to 1 if you have the `wcsrtombs' function.])
+                    ],
+                    [
+                      AC_MSG_RESULT([yes])
+                      AC_DEFINE_UNQUOTED([HAVE_WCSRTOMBS], 1, [Define to 1 if you have the `wcsrtombs' function.])
                     ]
                  )
 AC_MSG_CHECKING([for mbsrtowcs])
@@ -269,6 +273,10 @@
                     [
                       AC_MSG_RESULT([no])
                       AC_DEFINE_UNQUOTED([HAVE_MBSRTOWCS], 0, [Define to 1 if you have the `mbsrtowcs' function.])
+                    ],
+                    [
+                      AC_MSG_RESULT([yes])
+                      AC_DEFINE_UNQUOTED([HAVE_MBSRTOWCS], 1, [Define to 1 if you have the `mbsrtowcs' function.])
                     ]
                  )
 
